### PR TITLE
ENT-12497: Collaborative recovery

### PR DESCRIPTION
**Pull request by AnnaK-R3**
4.12 ENT release notes: https://anna.preview.docs.r3.com/en/platform/corda/4.12/enterprise/release-notes-enterprise.html#collaborative-recovery-removed

4.11 ENT release notes: https://anna.preview.docs.r3.com/en/platform/corda/4.11/enterprise/release-notes-enterprise.html#collaborative-recovery-deprecated

Standalone 4.12 Ledger Recovery folder with the Collab Recovery folder removed: https://anna.preview.docs.r3.com/en/platform/corda/4.12/enterprise/node/ledger-recovery/ledger-recovery.html

### CODE REVIEW

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md [hunk 1]
```diff
@@ -101,6 +101,10 @@ The [Archive Service]({{< relref "../../../../tools/archiving-service/archiving-
 
 Corda now supports JDK Azul 8u382 and Oracle JDK 8u381.
 
+### Collaborative Recovery deprecated
+
+The Collaborative Recovery solution, along with the associated CorDapps (LedgerSync and LedgerRecover), was deprecated in Corda 4.11 and will be removed in Corda 4.12. You must now use the new recovery tools introduced in version 4.11, as detailed in the following 4.11 release notes.
```
Comment (adelel1) [inline content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:106]: Slight update...

The Collaborative Recovery solution, along with the associated CorDapps (LedgerSync and LedgerRecover), is deprecated in Corda 4.11 and will be removed in Corda 4.12. You are advised to use the new recovery tools introduced in version 4.11, as detailed in the following 4.11 release notes.

File: content/en/platform/corda/4.12/enterprise/release-notes-enterprise.md [hunk 2]
```diff
@@ -198,6 +198,10 @@ The base Corda package includes several SDKs and libraries. These SDKs and libra
 | ledger-graph              | 1.3                           |
 | r3-tools                  | 4.12                          |
 
+#### Collaborative Recovery removed
+
+The Collaborative Recovery solution, along with the associated CorDapps (LedgerSync and LedgerRecover), was removed in Corda 4.12. You must now use the new recovery tools introduced in version 4.11, as described in the [Corda Enterprise Edition 4.11 release notes]({{< relref "../../4.11//enterprise/release-notes-enterprise.html#corda-enterprise-edition-411-release-notes-1" >}}).
```
Comment (adelel1) [inline content/en/platform/corda/4.12/enterprise/release-notes-enterprise.md:203]: should be .... is removed in Corda 4.12

**Comment by adelel1** [comment]
In the 4.11 documentation, in the Section Collaborative Recovery V1.2.1
Can you add the following text please as a stand out note, just so that anyone who didn't see the release note will see this section if they do directly to the doc.


The Collaborative Recovery solution, along with the associated CorDapps (LedgerSync and LedgerRecover), is deprecated in Corda 4.11 and will be removed in Corda 4.12. You are advised to use the new recovery tools introduced in version 4.11, as detailed in the 4.11 release notes.

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md (hunk 1 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:106]: updated

File: content/en/platform/corda/4.12/enterprise/release-notes-enterprise.md (hunk 2 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.12/enterprise/release-notes-enterprise.md:203]: updated

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**